tensorflow_addons/image/distance_transform_test.py

```diff
@@ -97,38 +97,37 @@ def test_image_with_invalid_dtype(self):
             # pylint: disable=bad-continuation
             with self.assertRaisesRegex(
                     TypeError, "`dtype` must be float16, float32 or float64"):
-                _ = distance_tranform_ops.euclidean_dist_transform(
+                _ = distance_transform_ops.euclidean_dist_transform(
                     image, dtype=output_dtype)
 
     def test_image_with_invalid_shape(self):
-        for invalid_shape in ([1], [2, 1], [2, 4, 4, 4, 1]):
-            image = tf.zeros(invalid_shape, tf.uint8)
-
-            # pylint: disable=bad-continuation
-            with self.assertRaisesRegex(
-                    ValueError, "`images` should have rank between 3 and 4"):
-                _ = distance_tranform_ops.euclidean_dist_transform(image)
-
         image = tf.zeros([2, 4, 3], tf.uint8)
         with self.assertRaisesRegex(ValueError,
                                     "`images` must have only one channel"):
-            _ = distance_tranform_ops.euclidean_dist_transform(image)
+            _ = distance_transform_ops.euclidean_dist_transform(image)
 
     def test_all_zeros(self):
-        image = tf.zeros([10, 10, 1], tf.uint8)
-        expected_output = np.zeros([10, 10, 1])
+        image = tf.zeros([10, 10], tf.uint8)
+        expected_output = np.zeros([10, 10])
 
         for output_dtype in [tf.float16, tf.float32, tf.float64]:
-            output = distance_tranform_ops.euclidean_dist_transform(
+            output = distance_transform_ops.euclidean_dist_transform(
                 image, dtype=output_dtype)
             self.assertAllClose(output, expected_output)
 
     def test_all_ones(self):
         image = tf.ones([10, 10, 1], tf.uint8)
-        output = distance_tranform_ops.euclidean_dist_transform(image)
+        output = distance_transform_ops.euclidean_dist_transform(image)
         expected_output = np.full([10, 10, 1], tf.float32.max)
         self.assertAllClose(output, expected_output)
 
+    def test_unknown_shape(self):
+        fn = distance_transform_ops.euclidean_dist_transform.\
```

Comment:
How about giving an alias for distance_transform_ops? In most cases, we'd better to avoid backlash because it's fragile as shown in the discussion https://www.python.org/dev/peps/pep-3125/

Reply:
I'm renaming `distance_transform_ops` -> `dist_ops`.